### ISSUE TITLE
Docs: check in CI that markdown is formatted properly

### ISSUE THREAD

**Issue by anton-trunov** [tact-lang/tact#1169]
This might require reformatting the existing files

**Comment by verytactical** [comment]
@novusnota This seems to be already fixed by `prettier`, isn't it?

**Comment by novusnota** [comment]
> [@novusnota](https://github.com/novusnota) This seems to be already fixed by `prettier`, isn't it?

Yeah, but only for the internal docs, since the `docs/` folder (public docs) is in `.prettierignore`. That was done long ago because current ruleset didn't suit them AFAIR.

**Comment by anton-trunov** [comment]
In the age of LLMs, can we please fix this issue already?
